src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerArgumentCollection.java

```diff
@@ -123,4 +124,15 @@ public ReadThreadingAssembler createReadThreadingAssembler() {
     @Argument(fullName = MAX_MNP_DISTANCE_LONG_NAME, shortName = MAX_MNP_DISTANCE_SHORT_NAME,
             doc = "Two or more phased substitutions separated by this distance or less are merged into MNPs.", optional = true)
     public int maxMnpDistance = getDefaultMaxMnpDistance();
+
+    @Advanced
+    @Argument(fullName = ALLELE_EXTENSION_LONG_NAME,
+            doc = "Likelihood and read-based annotations will only take in consideration reads " +
+                    "that overlap the variant or any base on further than this distance expressed in base pairs",
+            optional = true)
+    public int informativeReadOverlapRadius = 0;
```

Comment:
FYI in the last commit I change this value to 2.... not sure what the best value is... 0 seems to be the most parsimonious but it reduces QUAL and PL scores in integration test so keeping It to the current (for PLs) 2 is a more conservative approach.

Reply:
Let's keep it the same for now.  @jamesemery is overhauling the assembly engine and there's no point optimizing this parameter to the current excess of haplotypes.